packages/lab/src/overlay/OverlayPanelCloseButton.css

```diff
@@ -0,0 +1,15 @@
+.saltOverlayPanelCloseButton {
+  position: absolute;
+  right: 0;
+  z-index: 1;
+}
+
+.saltOverlayPanelCloseButton-button.saltButton-secondary {
+  position: sticky;
+  right: 0;
+  top: 0;
+}
+
+.saltOverlayPanelContent {
```

Comment:
This content spacing is only needed when the close button is present

Reply:
using !important doesn't seem like the ideal solution to override the padding when the close button is being used - thoughts ?

Reply:
yeah this isn't ideal because it applies on the site examples without a button

Reply:
updated to use the sibling selector instead, so will only apply to the specific overlay - but yeh `disableCloseButton` could be a prop, depends on what api we would prefer